ui/page/claim/internal/claimPageComponent/index.js

```diff
@@ -62,8 +61,6 @@ const select = (state, props) => {
     latestClaimUrl,
     preferEmbed,
     isResolvingUri: selectIsUriResolving(state, uri),
```

Comment:
Didn't forgot to update the view.jsx in here, the removed stuff already didn't existed in there.